include/paths.h

```diff
@@ -0,0 +1,5 @@
+#ifndef _PATHS_H_
+
+#define        _PATH_BSHELL        "/bin/sh"
```

Comment:
Is the extra spacing between define and _PATH_BSHELL intended? 

Can you also follow the header convention like this: https://github.com/ibmruntimes/zoslib/blob/main/include/locale.h#L1-L10